airbyte-integrations/base-java/Dockerfile

```diff
@@ -6,10 +6,10 @@ WORKDIR /airbyte
 COPY javabase.sh .
 
 # airbyte base commands
-ENV AIRBYTE_SPEC_CMD "./javabase.sh --spec"
-ENV AIRBYTE_CHECK_CMD "./javabase.sh --check"
-ENV AIRBYTE_DISCOVER_CMD "./javabase.sh --discover"
-ENV AIRBYTE_READ_CMD "./javabase.sh --read"
-ENV AIRBYTE_WRITE_CMD "./javabase.sh --write"
+ENV AIRBYTE_SPEC_CMD "/airbyte/javabase.sh --spec"
```

Comment:
discovered when running test with the worker. since the worker's working dir is the job root, we need an absolute path here. cool that these test caught that! 😄